app/controllers/crowdblog/posts_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Crowdblog::PostsController < ApplicationController
   before_action :get_categories, only: [:index, :show]
   def index
+    @page = params[:page] || '1'
```

Comment:
Is it really necessary to treat '1' as a string? shouldnt' it be transparent inside the controllers?